modules/ROOT/pages/migration/index.adoc

```diff
@@ -94,6 +238,91 @@ query {
 ----
 |===
 
+=== The bookmark field has been removed from the schema
+
+The bookmark field has been removed from the Mutation info response as it is no longer required.
+
+
+=== The `excludeDeprecatedFields` setting in the Neo4jFeaturesSettings has been changed
+
+As in version 6.x many of the deprecated fields have been removed, the `excludeDeprecatedFields` setting has been modified to reflect these changes.
+
+The following fields have been removed:
+
+
+  - `bookmark`
+  - `negationFilters`
+  - `arrayFilters`
+  - `stringAggregation`
+  - `aggregationFilters`
+  - `nestedUpdateOperationsFields`
+
+The following fields have been added:
+
+
+  - `implicitEqualFilters`
+  - `deprecatedOptionsArgument`
+  - `directedArgument`
+
+== Additions
+
+=== Added `count_EQ` as alternative to the deprecated count
+
+The `count_EQ` filter is now available as an alternative to the deprecated `count` filter.
+
+[cols="1,1"]
+|===
+|Before | Now
+
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(where: { actorsAggregate: { count: 10 } }) {
+    title
+  }
+}
+----
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(where: { actorsAggregate: { count_EQ: 10 } }) {
+    title
+  }
+}
+----
+|===
+
+=== Added the _EQ filter as alternative of the deprecated implicit equal filters
+
+The `_EQ` filter is now available as an alternative to the deprecated implicit equal filters.
```

Comment:
Here you've written `filters` while in count it is just `filter` which is the most correct?